.env

```diff
@@ -0,0 +1,2 @@
+ENVIRONMENT=development
+API_BASE_URL=https://catfact.ninja
```

Comment:
Should this be more realistic, a typical API endpoint I think runs at localhost:3000/api (or something)? 

cc @bbbneo333